src/BalconyBotanica/wwwroot/index.html

```diff
@@ -1,44 +1,81 @@
 <!DOCTYPE html>
 <html>
 
-<body>
-
-    <h1>Find your balcony plant</h1>
-
-
-    <form id="plantForm">
-        <h2><label for="sunlight">&#x27A1; Amount of sunlight</label></h2>
-        <input type="radio" id="sunlight" name="sunlight" value="FULL_SUN">Full sun <br />
-        <input type="radio" id="sunlight" name="sunlight" value="PART_SHADE">Part shade<br />
-        <input type="radio" id="sunlight" name="sunlight" value="FILTERED_SHADE">Filtered shade<br />
-        <input type="radio" id="sunlight" name="sunlight" value="FULL_SHADE">Full shade<br />
-
-        <h2><label for="spaceSize">&#x27A1; Space size</label></h2>
-        <input type="number" id="spaceSize" name="spaceSizeSquareMeters" value="m2">m2<br />
-
-        <h2><label for="watering">&#x27A1; Watering</label></h2>
-        <input type="radio" id="watering" name="wateringSchedule" value="MINIMUM">MINIMUM<br />
-        <input type="radio" id="watering" name="wateringSchedule" value="AVERAGE">AVERAGE<br />
-        <input type="radio" id="watering" name="wateringSchedule" value="FREQUENT">FREQUENT<br />
-
-        <h2><label for="function">&#x27A1; Plant function</label></h2>
-        <input type="radio" id="function" name="plantFunction" value="EATABLE">EATABLE<br />
-        <input type="radio" id="function" name="plantFunction" value="FLOWER">FLOWER<br />
-        <input type="radio" id="function" name="plantFunction" value="GREEN">GREEN<br />
-        <input type="radio" id="function" name="plantFunction" value="INSECTS">INSECTS<br />
-        <input type="radio" id="function" name="plantFunction" value="INSECTS">NONE<br />
-
-
-        <h2><label for="toxicity">&#x27A1; Toxicity</label></h2>
-        <input type="radio" id="toxicity" name="toxicity" value="POISONOUS_TO_HUMANS">POISONOUS_TO_HUMANS<br />
-        <input type="radio" id="toxicity" name="toxicity" value="POISONOUS_TO_PETS">POISONOUS_TO_PETS<br />
-        <input type="radio" id="toxicity" name="toxicity" value="NONE">NONE<br />
-        <br />
-        <br />
-
-        <button type="submit">&#x21AA; find my plant please </button>
+<body style="max-width: 600px;  margin: 0 auto; background-color: #AFE1AF;">
+    <div class="flower" style="position: relative; width: 100px; height: 100px; margin: 10px auto;">
+        <div class="petal"
+            style="position: absolute; width: 50px; height: 50px; background-color: #ff99cc; border-radius: 50%; top: 0; left: 25px;">
```

Comment:
I couldn't resist adding some css. I have now done it inline but that doesn't really make any sense.